Stratis.Bitcoin/MemoryPool/MempoolManager.cs

```diff
@@ -48,19 +52,51 @@ public List<TxMempoolInfo> InfoAll()
 			{
 				Trx = item.Transaction,
 				Time = item.Time,
-				FeeRate = new FeeRate(item.Fee, (int) item.GetTxSize()),
+				FeeRate = new FeeRate(item.Fee, (int)item.GetTxSize()),
 				FeeDelta = item.ModifiedFee - item.Fee
 			}).ToList();
 		}
 
+		internal async Task LoadPool(string fileName = null)
```

Comment:
Hi @Derrick- sorry I didn't have time to test this yet did you manage to get testnet fully synced?
One thing I noticed is that there are no checks for confirmed trx

If it was a while since the last mempool was persisted its likely many of the trx are already confirmed